<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_sua</class>
 <widget class="QDialog" name="Dialog_sua">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_sua_dia">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #2C7DA0, stop: 1 #4FBDBA
    );
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 20px;
    font: bold 15px &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #368DB2, stop: 1 #6FDCD1
    );
}
QPushButton:pressed {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #F06E6E, stop: 1 #FFC857
    );
    color: white;
}</string>
      </property>
      <property name="text">
       <string>Sửa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_huy_dia_fix">
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_huy_dia_fix {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #E53935, stop: 1 #EF5350
    );
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 22px;
    font: bold 15px &quot;Segoe UI&quot;;
}

QPushButton#pushButton_huy_dia_fix:hover {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #FF6F61, stop: 1 #FFA07A
    );
}

QPushButton#pushButton_huy_dia_fix:pressed {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #B71C1C, stop: 1 #D84315
    );
    color: #FFF;
}
</string>
      </property>
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>29</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_suadiem">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1.5px solid #B0BEC5;
    border-radius: 6px;
    padding: 6px 10px;
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
    selection-background-color: #A7E3F2;
}

QLineEdit:focus {
    border: 2px solid #2C7DA0;
    background-color: #FAFEFF;
    outline: none;
}

QLineEdit::placeholder {
    color: #90A4AE;
    font-style: italic;
}
</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_sualop">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1.5px solid #B0BEC5;
    border-radius: 6px;
    padding: 6px 10px;
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
    selection-background-color: #A7E3F2; 
}

QLineEdit:focus {
    border: 2px solid #2C7DA0;
    background-color: #FAFEFF;
    outline: none;
}

QLineEdit::placeholder {
    color: #90A4AE;
    font-style: italic;
}
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_suaten">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1.5px solid #B0BEC5;
    border-radius: 6px;
    padding: 6px 10px;
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
    selection-background-color: #A7E3F2; 
}

QLineEdit:focus {
    border: 2px solid #2C7DA0;
    background-color: #FAFEFF;
    outline: none;
}

QLineEdit::placeholder {
    color: #90A4AE;
    font-style: italic;
}
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_suaho">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1.5px solid #B0BEC5;
    border-radius: 6px;
    padding: 6px 10px;
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
    selection-background-color: #A7E3F2; 
}

QLineEdit:focus {
    border: 2px solid #2C7DA0;
    background-color: #FAFEFF;
    outline: none;
}

QLineEdit::placeholder {
    color: #90A4AE;
    font-style: italic;
}
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label__ho_dia_fix">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
}</string>
      </property>
      <property name="text">
       <string>Họ:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_ten_dia_fix">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
}</string>
      </property>
      <property name="text">
       <string>Tên:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_lop_dia_fix">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
}
</string>
      </property>
      <property name="text">
       <string>Lớp:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_diem_dia_fix">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
}</string>
      </property>
      <property name="text">
       <string>Điểm:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_suama">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1.5px solid #B0BEC5;
    border-radius: 6px;
    padding: 6px 10px;
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
    selection-background-color: #A7E3F2; 
}

QLineEdit:focus {
    border: 2px solid #2C7DA0;
    background-color: #FAFEFF;
    outline: none;
}

QLineEdit::placeholder {
    color: #90A4AE;
    font-style: italic;
}
</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ma_dia_fix">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font: bold 15px &quot;Segoe UI&quot;;
    color: #2C3E50;
}</string>
      </property>
      <property name="text">
       <string>Mã Sinh Viên:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
